pysqldb3/geopackage.py

```diff
@@ -411,7 +423,7 @@ def read_gpkg(self, dbo, table = None, gpkg_tbl = None, schema = None, port = 54
         :param private:
         :param gpkg_encoding: encoding of data within Geopackage
         :param print_cmd: Optional flag to print the GDAL command that is being used; defaults to False
-        :param bulk_upload: Default to False. Use read_gpkg_bulk_upload() to upload all gpkg tables to db.
+        :param bulk_upload: Default to False. Do not set to True; please use read_gpkg_bulk() to upload all gpkg tables to db.
```

Comment:
doesnt look like this is actually a param that can be set, which makes sense given the comment

Reply:
it doesnt hurt cause it just a doc string, but whats the point of saying not to do something that you cant do anyway